test/concerns/versions_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class VersionsTest < ActiveSupport::TestCase
+  test '_document_versions return empty hash for blank history' do
+    author = Author.create(name: 'Ann Gilbert')
+    assert_equal({""=>"--"}, author._document_versions)
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.